pkg/internal/genhelpers/generator.go

```diff
@@ -134,6 +134,7 @@ func generateAndSaveForAllObjects[T ObjectNameProvider, M GenerationModel](objec
 		}
 		filename := filenameProvider(s, model)
 		if err := WriteCodeToFile(&buffer, filename); err != nil {
+			log.Println("Failed to save: ", buffer.String())
```

Comment:
you can run the generator with dry-run and verbose options (check here: https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/pkg/internal/genhelpers/README.md#how-it-works) so this log is not necessary

Reply:
Yeah, my bad forgot to remove this.